### ISSUE TITLE
Ways to handle 'micro-scroll'

### ISSUE THREAD

**Issue by b-hub** [AndreiMisiukevich/CardView#52]
Hello, 
Currently I have a button inside a PanCardView, but it is very hard to register a tap on the button as moving your finger slightly will cause the page to begin to swipe (move with your finger). I see properties like 'SwipeThresholdDistance', but it doesn't seem to be having an effect. 

Is there a way to limit the swipe sensitivity to give an opportunity for the button to be clicked? This might just be an issue on Xamarin / Android, but if you can suggest something I can do with the CardView - I am all ears!

Thanks.

**Comment by AndreiMisiukevich** [comment]
Hi @b-hub and thanks for your feedback. I appreciate it.

Look at this method https://github.com/AndreiMisiukevich/CardView/blob/master/PanCardView.Droid/CardsViewRenderer.cs#L96

I think, we should add property for Android regarding to your request.. because SwipeThresholdDistance - it's distance, when cardsView understands, that it should start animation of navigating. So, you can move your cards slowly (And you will have to move 0.35 percent of your CardsView at least for changing card by default, but SwipeThresholdDistance allow to make switching card with less distance in case of swiping.

Hope it was clear =)

**Comment by b-hub** [comment]
Hi @AndreiMisiukevich Thank you for getting back so fast!

Looking at the code I can see that when SetIsTouchHandled is true -> OnInterceptTouchEvent will return true. I don't know enough about this, but by debugging I can see it prevents the tap event from firing. 

Do you have an idea to resolve this?

**Comment by AndreiMisiukevich** [comment]
I will debug it on weekend, i hope

**Comment by AndreiMisiukevich** [comment]
``` csharp
IsTouchHandled = Abs(xDelta) > Abs(yDelta) && Abs(xDelta) > 20;
```

Could you try to change this line as i wrote above and check the behavior

**Comment by b-hub** [comment]
Yes that works! So I think we could have another configurable property to hold the '20'?

**Comment by AndreiMisiukevich** [comment]
For sure) I will add it on this weekend
BTW, if you like my plugin, you may set stat on this repository :)

**Comment by AndreiMisiukevich** [comment]
Oh, and if you want, you can contribute to this project :)
Just fork my repo, create new bindable property with name Android{PropertyName} and default value 0, and handle this case at renderer :)

**Comment by AndreiMisiukevich** [comment]
Ok, cool, it will be available in the next release 1.5.0 =)

**Comment by AndreiMisiukevich** [comment]
MoveTresholdDistance property
